src/js/containers/covid19/awardSpendingAgency/AwardSpendingAgencyTableContainer.jsx

```diff
@@ -260,20 +259,25 @@ const AwardSpendingAgencyTableContainer = (props) => {
         }
     });
 
+    useEffect(() => {
+        if (Object.keys(spendingByAgencyTotals).length > 0 && resultsTotal) {
+            addUnlinkedData(results, resultsTotal, spendingByAgencyTotals);
+        }
+    }, [spendingByAgencyTotals, resultsTotal]);
 
     useEffect(() => {
```

Comment:
Considering in this section we are not keeping sort order through the different award types, we do not need this effect. Or we should have comments defining what is happening here.